### DISCUSSION TITLE
Welcome to Chiroyce's Discussions!

### DISCUSSION THREAD

**Discussion by Chiroyce1** [Chiroyce1/Chiroyce1#1] [Announcements]
## 👋 Welcome!
  You can use discussions to 
  * Ask questions you’re wondering about.
  * Share ideas.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Quantum-Codes** [comment]
Are you banned? (Crowbar asked me) also your comments are off (for being banned or some spammer?)

**Reply by imdevarsh** [reply]
@Chiroyce1 has replied

**Reply by Quantum-Codes** [reply]
Thank you!

**Comment by Chiroyce1** [comment]
Nope I'm not banned - someone was spamming and I had to close comments, that's all.

**Reply by Quantum-Codes** [reply]
Ok

**Reply by Quantum-Codes** [reply]
I'm didn't get notified for this message. Next time just use the reply feature :)

**Reply by Chiroyce1** [reply]
Yeah I used the wrong textbox lol

**Reply by Chiroyce1** [reply]
@Quantum-Codes So what do you mainly do these days? Do you still have a chance of getting your Scratch account unbanned?

**Reply by Quantum-Codes** [reply]
I have lost hope on getting unbanned.i will send another appeal tomorrow I guess. I think I am blacklisted or smth.. 
![Screenshot_20220111-193451_Chrome](https://user-images.githubusercontent.com/87054411/150278499-6d6a878b-d4ee-49c6-876b-43775c0b4371.jpg)


Now I'm working on my python package called ScraGet https://github.com/Quantum-Codes/ScraGet

**Reply by Quantum-Codes** [reply]
Oh yeah... I was trying github  actions too!

**Reply by Chiroyce1** [reply]
nice nice

**Reply by Quantum-Codes** [reply]
I just made this animated slider https://CSS-testings.quantumcodes.repl.co (by referring your site :P)

**Reply by Chiroyce1** [reply]
It looks great!!

**Reply by Quantum-Codes** [reply]
Ty!

**Comment by Sid72020123** [comment]
Hi Chiroyce! Welcome to Github!

**Reply by Chiroyce1** [reply]
Thanks! 😀

**Comment by ghost** [comment]
Hi, Chiroyce!

**Reply by Chiroyce1** [reply]
Hello!

**Reply by ghost** [reply]
Thanks for replying :)

**Reply by Chiroyce1** [reply]
So what are you working on now? Are you planning to come back to Scratch?

**Reply by ghost** [reply]
I'm working on Rudra, you can check my profile, and I don't know if I'll be back on scratch, I'm uncertain.

**Comment by Quantum-Codes** [comment]
Why's it that whenever I say someone "I see your password in code, change it", people take it **negatively** and never end up removing from their code :(

**Reply by Chiroyce1** [reply]
bruh why do they do that? :(

**Reply by Quantum-Codes** [reply]
Idk

**Reply by ghost** [reply]
Wait... how do you see their password?

**Reply by Chiroyce1** [reply]
In [Replit](https://replit.com), if you have the free plan, your code will be publicaly visible. So when using libraries like [scratchclient](https://github.com/CubeyTheCube/scratchclient), you have to use environment variables to hide your password, but some people put it right in their code.

**Comment by da-Jester** [comment]
Hello! I'm -Krios- on scratch.

**Reply by Chiroyce1** [reply]
Hello!
